CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+## [0.56.12](https://github.com/brave/brave-browser/releases/tag/v0.56.12)
+
+- Updated the Brave shields panel. ([#1339](https://github.com/brave/brave-browser/issues/1339))
```

Comment:
Maybe this is too generic, maybe say Updated the Brave shields panel with a new look